.github/workflows/pythonpackage.yml

```
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8, 3.9, "3.10", 3.11]
```

Not sure what the double quotes are for?

```suggestion
        python-version: [3.8, 3.9, 3.10, 3.11]
```

---

Yaml otherwise converts 3.10 to 3.1 :disappointed: